contracts/HATVaults.sol

```diff
@@ -93,7 +95,7 @@ contract  HATVaults is Governable, HATMaster {
     //pid -> SubmittedClaim
     mapping(uint256 => SubmittedClaim) public submittedClaims;
     //poolId -> (address -> requestTime)
-    mapping(uint256 => mapping(address => uint256)) public withdrawRequests;
+    mapping(uint256 => mapping(address => uint256)) public WithdrawEnableStartTime;
```

Comment:
linter warns that this should be `withdrawEnableStartTime` (i.e. lower case)

Reply:
oh right my bad